pdata/pcommon/common.go

```diff
@@ -648,6 +664,7 @@ func (m Map) RemoveIf(f func(string, Value) bool) {
 //
 // Important: this function should not be used if the caller has access to
 // the raw value to avoid an extra allocation.
+// NOTE: The method will be deprecated in 0.60.0. Use Get and CopyTo instead.
```

Comment:
Get and UpsertEmpty?

Reply:
I added more descriptive guidelines to the deprecation comments. PTAL